test/fixtures/output/network/private-link/attachment/describe-azure.golden

```diff
@@ -0,0 +1,10 @@
++--------------------------------+-------------------------+
+| ID                             | platt-azure             |
+| Name                           | azure-platt             |
```

Comment:
These are so similarly named it's confusing which one is the ID and which one is the name... maybe rename to the following:
```suggestion
| ID                             | platt-azure                      |
| Name                           | my-azure-private-link-attachment |
```